packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -308,7 +314,14 @@ export const CommentsItem = ({ treeOptions, comment, nestingLevel=1, isChild, co
               collection={Comments}
               hideKarma={post?.hideCommentKarma}
             />
-
+            {(votingSystem === 'TwoFactorAgree') && <div className = {classes.agreeButtons}>
```

Comment:
This is wrong. All of the vote-buttons on a single comment need to be sharing a single `useVote` call; what you should have done was put the second pair of buttons inside of `SmallSideVote`, not add a second usage of the `SmallSideVote` component. Doing it this way means that the buttons don't have a unified state, which is going to cause problems in client-side updating.